module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -276,20 +277,23 @@ protected function getItemStatus(
     ) {
         // Summarize call number, location and availability info across all items:
         $callNumbers = $locations = [];
-        $use_unknown_status = $available = false;
+        $useUnknownStatus = false;
+        $available = 'false';
         $services = [];
+        $reserve = !empty($record);
 
         foreach ($record as $info) {
             // Find an available copy
-            if ($info['availability']) {
-                $available = true;
+            if ($info['availability'] && 'true' !== $available) {
+                $available = $this->availabilityToString($info['availability']);
             }
             // Check for a use_unknown_message flag
-            if (
-                isset($info['use_unknown_message'])
-                && $info['use_unknown_message'] == true
-            ) {
-                $use_unknown_status = true;
+            if ($info['use_unknown_message'] ?? false) {
+                $useUnknownStatus = true;
+            }
+            // Check for non-reserves:
+            if ($info['reserve'] === 'N') {
+                $reserve = false;
```

Comment:
Is there a reason we assume things are on reserve unless otherwise specified? Might this lead to false positives?

Reply:
I've no idea what I was thinking. Now inverted.